io.gitlab.adhami3310.Footage.json

```diff
@@ -0,0 +1,259 @@
+                        "http://ftp.debian.org/debian/pool/main/v/vo-aacenc/vo-aacenc_0.1.3.orig.tar.gz"
+                    ],
+                    "sha256": "e51a7477a359f18df7c4f82d195dab4e14e7414cbd48cf79cc195fc446850f36"
+                }
+            ]
+        },
+        {
+            "name": "svt-av1",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DBUILD_SHARED_LIBS=ON",
+                "-DBUILD_APPS=OFF",
+                "-DBUILD_DEC=ON",
+                "-DBUILD_ENC=ON",
+                "-DBUILD_TESTING=OFF"
+            ],
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.com/AOMediaCodec/SVT-AV1",
+                    "commit": "ea296ef350714fb6f105b420fb0bc321d9997ffd"
+                }
+            ]
+        },
+        {
+            "name": "gstreamer-svt-av1",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cd gstreamer-plugin && meson -Dprefix=/app build",
+                "cd gstreamer-plugin && ninja -C build install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/master/SVT-AV1-master.tar.gz?path=gstreamer-plugin",
+                    "sha256": "19a9f37baec61b805964a42e0123cd09838a27875f439d0336418fb1dbe7f524"
+                }
+            ]
+        },
+        {
+            "name": "gstreamer",
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": [
+                "--libdir=lib",
+                "--wrap-mode=nodownload",
+                "-Dgpl=enabled",
+                "-Dgood=enabled",
+                "-Dbase=enabled",
+                "-Dugly=enabled",
+                "-Dvaapi=enabled",
+                "-Ddoc=disabled",
+                "-Dgst-examples=disabled",
+                "-Dpython=disabled"
+            ],
+            "post-install": [
+                "mkdir -p /app/lib/codecs/lib/gstreamer-1.0",
+                "mv /app/lib/libvo-aacenc.so* /app/lib/codecs/lib",
+                "mv /app/lib/libx265.so* /app/lib/codecs/lib",
+                "mv /app/lib/libfdk-aac.so* /app/lib/codecs/lib",
+                "mv /app/lib/gstreamer-1.0/libgst{fdkaac,voaacenc,x265}.so /app/lib/codecs/lib/gstreamer-1.0",
+                "mkdir -p /app/lib/codecs/lib",
+                "mv /app/lib/gstreamer-1.0/libgstlibav.so /app/lib/codecs/lib/gstreamer-1.0"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "tag": "1.22.3",
+                    "url": "https://gitlab.freedesktop.org/gstreamer/gstreamer.git",
```

Comment:
do you really need 1.22?

Reply:
I believe 1.20 is missing one feature that is used in the application. It complained about videoconvertscale not being available last time.